From 614dbbdcda54e02bd83c4198e1cf5b4cebd43584 Mon Sep 17 00:00:00 2001
From: Kever Yang <kever.yang@rock-chips.com>
Date: Mon, 10 Aug 2020 11:40:12 +0800
Subject: [PATCH] input: remotectl: Set REMOTECTL driver as tristate

The removectl drivers may be build as driver, set Kconfig to tristate to
support it.

Signed-off-by: Kever Yang <kever.yang@rock-chips.com>
Change-Id: Id786747e0eb0748644dc0fe76110e2805d4f2500
---
 drivers/input/remotectl/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/input/remotectl/Kconfig b/drivers/input/remotectl/Kconfig
index b1c9364b3812..67558b48869e 100644
--- a/drivers/input/remotectl/Kconfig
+++ b/drivers/input/remotectl/Kconfig
@@ -3,7 +3,7 @@
 # Input remotectl driver configuration
 #
 menuconfig ROCKCHIP_REMOTECTL
-	bool "rockchip remotectl"
+	tristate "rockchip remotectl"
 	default n
 	help
 	  Say Y here, will suport rk remotectl.
@@ -13,7 +13,7 @@ menuconfig ROCKCHIP_REMOTECTL
 if ROCKCHIP_REMOTECTL
 
 config ROCKCHIP_REMOTECTL_PWM
-	bool "rockchip remoctrl pwm capture"
+	tristate "rockchip remoctrl pwm capture"
 	default n
 
 endif
-- 
2.35.3

